### PR TITLE
feat(readme): add discord badge to readme

### LINKED ISSUE #9712: Add a link to the Deno Discord in the Readme
I'm happy to make a pull request if this is wanted. Maybe a button at the top next to the Twitter one?

### PR DESCRIPTION
**Pull request by makuzaverite**
fix #9712

### CODE REVIEW

**Comment by makuzaverite** [comment]
I'm using sheilds.io badges I'm facing error that widgets aren't enabled. the server admin

image

File: README.md [hunk 1]
```diff
@@ -90,3 +91,5 @@ To contribute, please read our
 [Build status]: https://github.com/denoland/deno/actions
 [Twitter badge]: https://twitter.com/intent/follow?screen_name=deno_land
 [Twitter handle]: https://img.shields.io/twitter/follow/deno_land.svg?style=social&label=Follow
+[Discord badge]: https://img.shields.io/discord/442252698964721669?label=Discord&style=social
+[Discord server]: https://discord.gg/GSSz635q7F
```
Comment (satyarohith) [inline README.md:95]:
```suggestion
[Discord server]: https://discord.gg/deno
```

**Comment by mjrlowe** [comment]
@makuzaverite I think you just need to format the readme so that it passes the lint test, and then this pull request should be able to be merged.

**Comment by makuzaverite** [comment]
@mjrlowe  how then ?

**Comment by mjrlowe** [comment]
Try running `deno fmt`?

**Comment by makuzaverite** [comment]
Moved here #9846

**Pull request closed without merging**